src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -165,10 +165,23 @@ def __call__(
                       np.array}` with optionally a `"stride": (left: int, right: int)` than can ask the pipeline to
                       treat the first `left` samples and last `right` samples to be ignored in decoding (but used at
                       inference to provide more context to the model). Only use `stride` with CTC models.
+            return_timestamps (*optional*, `str`):
+                Only available for pure CTC models. If set to `"char"`, the pipeline will return `timestamps` along the
+                text for every character in the text. For instance if you get `[{"text": "H", "timestamps": (0.5,0.6),
```

Comment:
(nit) but our models usually never have both capitalized and non-capitalized letters
```suggestion
                text for every character in the text. For instance if you get `[{"text": "h", "timestamps": (0.5,0.6),
```